sympy/algebras/tests/test_quaternion.py

```diff
@@ -79,7 +84,7 @@ def test_quaternion_functions():
     assert integrate(Quaternion(x, x, x, x), x) == \
     Quaternion(x**2 / 2, x**2 / 2, x**2 / 2, x**2 / 2)
 
-    assert Quaternion.rotate_point((1, 1, 1), q1) == (1 / 5, 1, 7 / 5)
+    assert Quaternion.rotate_point((1, 1, 1), q1) == (Rational(1,5) , 1 , Rational(7,5))
```

Comment:
un-related change, don't make this change here.

Reply:
It would be good if I do that in a separate PR? @gxyd

Reply:
Yes, you make a separate PR for any other changes, which are not related to the issue you are solving.